docs/production-deployment/cloud/service-health.mdx

```diff
@@ -11,41 +11,38 @@ tags:
   - health
 ---
 
-Temporal Cloud metrics help you to monitor the Service health of your production deployments.
+Temporal Cloud metrics help you to monitor the Service health of your production deployments. This documentation addresses some common best practices for monitoring Service health.
 
-## Monitor availability issues
+## Monitoring availability issues
```

Comment:
This is a SEO decision that I believe @napcs signed off on and I know Patrick and Cully adopted. We dropped present participles to make our headlines more friendly to search.

Since @jsundai was involved in the IA work, she probably has more insight into the specifics.

In addition, we started adding the question form with reworded content using bolding after the H2, for example:

```
## Monitor availability issues

**How do I detect potential problems with the Temporal Cloud service that might affect my Workflows?**
```

Again, check with Jwahir and Brian for specifics